src/libs/ReportUtils.js

```diff
@@ -893,6 +894,7 @@ function buildOptimisticCreatedReportAction(ownerEmail) {
             ],
             automatic: false,
             sequenceNumber: 0,
+            reportActionID: optimisticReportActionID,
```

Comment:
There's no need for the `optimisticReportActionID` variable. Also, I think we're trying to move away from that `optimistic*ID` naming convention since it's not just the optimistic ID, it's **the** ID.

Reply:
In the same vein, should we rename this function to `buildCreatedReportAction`?

Reply:
>There's no need for the optimisticReportActionID variable. 

Yes we need it https://github.com/Expensify/App/pull/13324#discussion_r1042334498

Reply:
We can grab it from the `optimisticReportAction` right? That was mentioned here https://github.com/Expensify/App/pull/13324#discussion_r1042203123. Is there some other use that I'm missing?

Reply:
My bad, I thought this was another part of the code 🤦 You're right, updated.